aiokafka/consumer/consumer.py

```diff
@@ -998,7 +998,6 @@ def getmany(self, *partitions, timeout_ms=0, max_records=None):
         return records
 
     if PY_35:
-        @asyncio.coroutine
```

Comment:
This may break on early Python3.5. Probably a minor version check should be present

Reply:
Addressed in followup beeb9f07074dee23d8a8d4184f2b81df0088ca31